storage/remote/queue_manager.go

```diff
@@ -366,29 +366,29 @@ type QueueManager struct {
 	quit        chan struct{}
 	wg          sync.WaitGroup
 
-	dataIn, dataDropped, dataOut, dataOutDuration *ewmaRate
+	DataIn, DataDropped, DataOut, DataOutDuration *EWMARate
```

Comment:
I think these should be made available via function, if you need them, rather than by exporting just these three members.